src/editor/components/menu/view-mode-toggle-button.jsx

```diff
@@ -16,9 +16,17 @@ export class ViewModeToggleButtonUnconnected extends React.Component {
   };
```

Comment:
We probably would want to factor this into its own higher-level component, as it's not really a button anymore.

There are other minor UI Chrome elements that we want in the report view (like a spinner icon when the document is loading) so this might be an opportunity to pave the way for that.